src/stories/Page/Page.tsx

```diff
@@ -11,7 +11,7 @@ type User = {
     children?: any
 }
 
-export const Page: React.VFC = (children: any) => {
+export const Page: React.VFC = () => {
```

Comment:
Looks like React.VFC is deprecated ;) Any chance we can get rid of VFC/FC altogether?

Reply:
Yes, can take measures as pointed out in this article:

https://www.mydatahack.com/moving-away-from-fc-and-vfc/